config/clusters/catalystproject-latam/unam.values.yaml

```diff
@@ -23,3 +23,87 @@ jupyterhub:
         admin_users:
           - nselem
           - miguel-mx
+  singleuser:
+    profileList:
+      - display_name: Jupyter SciPy Notebook
+        description: Python environment
+        slug: jupyter
+        default: true
+        kubespawner_override:
+          image: quay.io/jupyter/scipy-notebook:2024-03-04
+          default_url: /lab
+        profile_options: &profile_options
+          resource_allocation: &resource_allocation
+            display_name: Resource Allocation
+            choices:
+              mem_0_7:
+                display_name: Up to 2G of RAM and 1 CPU
+                kubespawner_override:
+                  mem_guarantee: 732421K
+                  mem_limit: 2G
+                  cpu_guarantee: 0.093
+                  cpu_limit: 1
+              mem_3_0:
+                display_name: Up to 6G of RAM and 2 CPUs
+                kubespawner_override:
+                  mem_guarantee: 2929687.5K
+                  mem_limit: 6G
+                  cpu_guarantee: 0.375
+                  cpu_limit: 2
+              mem_12_0:
+                display_name: Up to 24G of RAM and 3 CPUs
+                kubespawner_override:
+                  mem_guarantee: 11718750K
+                  mem_limit: 24G
+                  cpu_guarantee: 1.5
+                  cpu_limit: 3
+              # guarantee = limit because we are requesting a complete node, no space for other user
+              # this was generated with deploy generate command
+              # TODO: this was generated with deploy generate command, but with mem -= 1024 * 1024,
+              #       otherwise the pod startup fail with "insufficient memory"
+              mem_116_0:
+                display_name: Up to 116.0 GB RAM and 15.4 CPUs
+                kubespawner_override:
+                  mem_guarantee: 124555718656
```

Comment:
Given @consideRatio's concerns in https://github.com/2i2c-org/infrastructure/pull/4131#pullrequestreview-2081777499, and the fact that we want to treat this as an issue of size '1', I think we shouldn't try to fiddle with the resource allocation script as part of this PR - especially given that the catalyst project hubs don't currently use it anyway!

Given that, I suggest the following values for the profiles:

1. `Up to 115GB RAM and 15 CPUs`, with mem_guarantee and mem_limit set to just `115G`, and cpu_guarantee and cpu_limit set to `15`. This should give us enough 'headroom' to not need tweaking for a while. This goes on the `n2-highmem-16` nodes
2. `Up to 486 GB RAM and 63 CPUs`, with mem_guarantee and mem_limit set to `486GB` and cpu_guarantee and cpu_limit set to `63`. Again, this should give us enough headroom to not need tweaking for a while. This goes on `n2-highmem-64` nodes.

This would allow us to service this request and get the community going, and not need everything to be 'perfect'. Does that seem ok to you, @AIDEA775?
4.

Reply:
Okay! That sounds good!